src/coreclr/jit/emitarm64.cpp

```diff
@@ -1087,6 +1087,22 @@ bool emitter::emitInsMayWriteToGCReg(instrDesc* id)
         case IF_SR_1A: // SR_1A   ................ ...........ttttt      Rt       (dc zva, mrs)
             return ins == INS_mrs_tpid0;
 
+        // Below SVE instructions write to GPR and hence GC reg
```

Comment:
How did you figure out this list? Just want to make sure we've got all of them

Reply:
I just went through the `instrsarm64sve.h` and went over the instructions that has GPR as destination register and added them to this list.